.github/workflows/npm-publish.yml

```
@@ -0,0 +1,51 @@
+name: npm-publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      package-name:
+        description: Package name
+        type: choice
+        default: "couper"
+        required: true
+        options:
+        - "couper"
+        - "@avenga/couper"
+      package-version:
+        description: Package version
+        required: true
+        default: ${{ GITHUB_REF_NAME }}
```

```suggestion
        default: ${{ github.ref_name }}
```
no idea if it would work with the web-ui

---

No idea either :smile: